inst/pages/exercises.qmd

```diff
@@ -1560,11 +1560,14 @@ sechm(
 #| code-summary: "Show solution"
 #| eval: false
 
+library(mia)
 library(pheatmap)
 
-# Create example data
-set.seed(123)
-example_data <- matrix(rnorm(200), nrow = 20, ncol = 10)
+# Load example dataset from mia package
+data(enterotype)
+
+# Extract the data matrix for heatmap
+example_data <- assay(enterotype, "counts")
```

Comment:
Is it necessary the extract the data matrix, or could we operate on the tse data object directly using sechm?

sechm provides a simple interface between SummarizedExperiment objects and the ComplexHeatmap package. It enables plotting annotated heatmaps from SE objects, with easy access to rowData and colData columns, and implements a number of features to make the generation of heatmaps easier and more flexible. 
https://www.bioconductor.org/packages/release/bioc/html/sechm.html

Reply:
This comment applies regardless of ComplexHeatmap or sechm